WBooks/Extensions/Images.swift

```diff
@@ -58,4 +58,29 @@ extension UIImage {
         return UIImage(named: "tabBarSettingsUnselect")!
     }
     
+    static var bookDefault: UIImage {
+        return UIImage(named: "imageBookMock")!
+    }
+    
+}
+
+extension UIImageView {
+    
+    public func imageFromUrl(urlString: String) {
+        if let url = URL(string: urlString.replacingOccurrences(of: "http", with: "https")) {
```

Comment:
Why we change http to https?

Reply:
If I do not make the change I get the following error,

![Screen Shot 2021-08-25 at 9 52 18 AM](https://user-images.githubusercontent.com/88329934/130813527-4da71a5e-ceab-46d2-b224-32062e609920.png)

since the url supports https, I preferred to change it, in order not to modify the info.plist so that it admits insecure urls